styles/main.css

```diff
@@ -0,0 +1,781 @@
+@font-face {
+  font-family: "COCOGOOSE";
+  font-style: normal;
+  font-weight: 400;
+  src: local("COCOGOOSE"), url("https://fonts.cdnfonts.com/s/26936/cocogoose_trial.woff") format("woff");
+}
+
+@font-face {
+  font-family: "Lato";
+  font-style: normal;
+  font-weight: 400;
+  src: url(https://fonts.gstatic.com/s/lato/v22/S6uyw4BMUTPHjx4wXg.woff2)format("woff2");
+}
```

Comment:
The pages use the same exact fonts from the original design 💪 Nice!

Reply:
@mateo951 Thank you.